hal/tests/TESTS/pin_names/generic/main.cpp

```diff
@@ -26,7 +26,7 @@ Requirements specified in docs/design-documents/hal/0004-pin-names-general-guide
 
 #ifndef LED1
 #error [NOT_SUPPORTED] Target is not following mbed-os pin names standard // Test is set as Skipped
-// #error [NOT_SUPPORTED] Target is not following mbed-os pin names standard // Test is set as Error
+// #error Target is not following mbed-os pin names standard // Test is set as Error
```

Comment:
shall we just remove this line completely?

Reply:
During this test creation, idea was to set this test as error as pin names standard is mandatory.

After discussion with @MarceloSalazar test has been set as skipped (NOT_SUPPORTED) waiting for targets migration.

Reply:
there is an error (line 28) that we can fix later. we do not need to have this commented out code (line 29) ?

Reply:
Done